src/com.company/backtracking/PalindromePartitioning.java

```diff
@@ -0,0 +1,38 @@
+package backtracking;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PalindromePartitioning {
+
+    void palindromeParitioning(String string, int index, List<String> currentList, List<List<String>> result) {
```

Comment:
Add access specifier for all the methods.

Reply:
add for all the methods